css/main.css

```diff
@@ -34,6 +34,60 @@ label.dynamic-info {
 	font-style: italic;
 }
 
+input[type='number'] {
+	max-width: 7em;
+	/* Remove spinner controls from Firefox */
+	-moz-appearance: textfield;
+}
+/* Re-applies the controls on :hover and :focus
+to get the same behavior as on chrom* */
+input[type='number']:hover,
+input[type='number']:focus {
+	-moz-appearance: number-input;
+}
+
+/* Enables the first input's help text to pop
+over the lasts of this group.
+Also keeps related inputs together. */
+fieldset.no-space-wrap {
```

Comment:
`.no-space-wrap` is not semantic. Why not simply style the `fieldset` itself, if indeed what you want is to group a set of fields together?